File: BasicEncrypter.py
print("This program takes in a string and encrypts it," +
      " or shifts all letters two positions down the alphabet. \n" + 
      "For example, 'A' will become 'C', 'D' will become 'F', " +
      " and so on. \n At the end of the alphabet, 'X' becomes 'A' and " +
      " 'Z' becomes 'B'. \n Non-letters (numbers, symbols, etc.)" +
      " will not be shifted at all. \n \n")
print("Enter your 5-letter word to encrypt:")
word =

letter1 = 
letter2 = 
letter3 = 
letter4 = 
letter5 = 

print("The encrypted word:", end=' ')
encryptLetterAndPrint(letter1)
encryptLetterAndPrint(letter2)
encryptLetterAndPrint(letter3)
encryptLetterAndPrint(letter4)
encryptLetterAndPrint(letter5)

print()


# Don't worry about this code, it just encrypts a letter.
def encryptLetterAndPrint(word):
    letter = word[0]
    
    if ('A' <= letter <= 'Z') or ('a' <= letter <= 'z'):
        letter = chr(ord(letter) + 2)
        if (('A' < letter < 'a') or (letter > 'Z' and letter > 'z')):
            letter = chr(ord(letter) - 26)
    print(letter, end='')

File: BasicEncrypter.java
public class BasicEncrypter {
    public static void main(String[] args) {
        System.out.println("This program takes in a string and encrypts it," +
            " or shifts all letters two positions down the alphabet. \n" + 
            "For example, \'A\' will become \'C\', \'D\' will become \'F\', " +
            " and so on. \n At the end of the alphabet, \'X\' becomes \'A\' and " +
            " \'Z\' becomes \'B\'. \n Non-letters (numbers, symbols, etc.)" +
            " will not be shifted at all. \n \n");
        System.out.println("Enter your 5-letter word to encrypt:");
        
        String letter1 = 
        String letter2 = 
        String letter3 = 
        String letter4 = 
        String letter5 = 
        
        System.out.println("The encrypted word:");
        encryptLetterAndPrint(letter1);
        encryptLetterAndPrint(letter2);
        encryptLetterAndPrint(letter3);
        encryptLetterAndPrint(letter4);
        encryptLetterAndPrint(letter5);
        
        System.out.println();
    }
    
    
    
    
    
    
    
    
    
    
    
    
    /**
     * NOTE TO STUDENTS: You do not have to worry about this code. All
     * this function does is do the encryption of ONE letter. You may try to
     * understand it if you want, but it is not necessary to edit 
     * this function in any other way.
     */
    public static void encryptLetterAndPrint(String word) {
        char letter = word.charAt(0);
        
        if (letter >= 'A' && letter <= 'Z' || letter >= 'a' && letter <= 'z') {
            letter += 2;
            if ((letter >'Z' && letter < 'a') || (letter > 'Z' && letter < 'a' || letter > 'z'))
                letter -= 26;
        }
        System.out.print(letter);
    }
}
